authors/apps/article/views.py

```diff
@@ -1,21 +1,17 @@
 from django.db.utils import IntegrityError
-from django.shortcuts import get_object_or_404
 from rest_framework.permissions import IsAuthenticated, IsAuthenticatedOrReadOnly
 from rest_framework.response import Response
-from rest_framework import status
```

Comment:
I assume that these deletions are on unused code, right?

Reply:
@JoanNgatia yes. These imports were being repeated several time others are imports that are not being used

Reply:
cool, thanks. This PR is good to go